pkg/console/controllers/clidownloads/controller_test.go

```diff
@@ -124,6 +124,14 @@ The oc binary offers the same capabilities as the kubectl binary, but it is furt
 							Href: "https://www.example.com/amd64/linux/oc.tar",
 							Text: "Download oc for Linux for x86_64",
 						},
+						{
+							Href: "https://www.example.com/amd64/linux/oc.rhel8.tar",
```

Comment:
Hey @jhadvig 😄 Since they all say "Download oc for ..." can we move that to the sentence that comes before the bullets to say something like 
"The oc binary offers the same capabilities as the kubectl binary, but it is further extended to natively support OpenShift Container Platform features. You can download oc using the following links."
We can also specify that last sentence further if you prefer.

Reply:
Yeah that looks good to me to keep consistency! Thanks for looping me in!!